packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -343,28 +344,35 @@ export interface ITelemetryContext {
 	setMultiple(
 		prefix: string,
 		property: string,
-		values: Record<string, TelemetryBaseEventPropertyType>,
+		values: Record<string, TelemetryEventPropertyTypeExt>,
 	): void;
+}
 
+/**
+ * Contains telemetry data relevant to summarization workflows.
+ * This object is expected to be modified directly by various summarize methods.
+ * @public
```

Comment:
This used to be alpha, we don't need to make it public, right?

Reply:
Right right. Modified that by mistake. Thanks for correcting it!